test/test_helper.exs

```diff
@@ -1 +1,2 @@
 ExUnit.start()
+ExUnit.configure(exclude: [:integration])
```

Comment:
You can pass these options to `ExUnit.start`.

Reply:
🤦‍♂️